datashuttle/command_line_interface.py

```diff
@@ -1036,6 +1037,9 @@ def main() -> None:
     """
     args = parser.parse_args()
 
+    if args.project_name in ["tui", "gui"]:
```

Comment:
Nice hack.

However, whenever I call `datashuttle tui` or `datashuttle gui`, I get the following warning:

> UserWarning: Configuration file has not been initialized. Use make_config_file() to setup before continuing.
  warnings.warn(
> Datashuttle project: tui. Add additional commands, see --help for details

The TUI launches as expected, but this warning is visible after I close it (and it may confuse users)